community/server/src/main/java/org/neo4j/server/modules/RESTApiModule.java

```diff
@@ -69,8 +71,9 @@ public void start()
     {
         URI restApiUri = restApiUri( );
 
-        webServer.addFilter( new CollectUserAgentFilter( clientNames() ), "/*" );
-        webServer.addFilter( new CorsFilter( logProvider ), "/*" );
+        webServer.addFilter( new CollectUserAgentFilter( clientNames() ), "/*", Collections.emptyMap() );
+        webServer.addFilter( new CorsFilter( logProvider ), "/*", Collections.singletonMap(
+                "access_control_allow_origin", config.get( access_control_allow_origin ) ) );
```

Comment:
What is the reason for piping this as an untyped config map thing, rather than passing the `access_control_allow_origin` setting directly to the constructor of the CorsFilter?